### ISSUE TITLE
GVim + cmd.exe - Multiprocessing Fails

### ISSUE THREAD

**Issue by starcraftman** [starcraftman/vim-plug#1]
For this python implementation, I'm using multiprocessing for parallelism. If this can't be resolved, we'll just not support on Windows (even though that annoys me). There must be a bug tracking this with Vim somewhere.

Error:
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Dec 8 2014 20:34:48)
Unknown option argument: "--multiprocessing-fork"
More info with: "vim -h"

I know this has to do with Windows not having fork, info on this at: 
https://groups.google.com/forum/#!searchin/vim_dev/multiprocessing/vim_dev/cZeWKa3w8DU/p8_kdi7WAfUJ
http://stackoverflow.com/questions/15636266/embedded-python-multiprocessing-not-working

**Comment by starcraftman** [comment]
Note on possible design solution, requires a fair sized refactor.
- Dump multiprocessing.
- Make main thread write a stub file to be executed via commandline. Reads in a pickle file for input & simply prints results to output. (i.e. stub executed by sub command is ./thread.py fname).
- Output captured by a Popen call, much the same as the threads now work. Loop so that many Popen's running in parallel.
- Main thread monitors and updates buf as Popens finish.

**Comment by starcraftman** [comment]
Working on this in the branch use_subprocess.

**Comment by starcraftman** [comment]
Above comment useless now. New approach is normal threads with queue + lock. Should work seamlessly. Already merged to main.

**Comment by starcraftman** [comment]
No longer relevant issue.
